deckhouse-controller/pkg/controller/module-controllers/release/updater.go

```diff
@@ -41,21 +41,21 @@ func newModuleUpdater(logger logger.Logger, nConfig *updater.NotificationConfig,
 	kubeAPI updater.KubeAPI[*v1alpha1.ModuleRelease], enabledModules []string) *updater.Updater[*v1alpha1.ModuleRelease] {
 	return updater.NewUpdater[*v1alpha1.ModuleRelease](logger, nConfig, mode,
 		updater.DeckhouseReleaseData{}, true, false, kubeAPI, newMetricsUpdater(),
-		newSettings(), newWebhookDataGetter(), enabledModules)
+		newSettings(), newWebhookDataSource(), enabledModules)
 }
 
-func newWebhookDataGetter() *webhookDataGetter {
-	return &webhookDataGetter{}
+func newWebhookDataSource() *webhookDataSource {
+	return &webhookDataSource{}
 }
 
-type webhookDataGetter struct {
+type webhookDataSource struct {
 }
 
-func (w *webhookDataGetter) GetMessage(release *v1alpha1.ModuleRelease, releaseApplyTime time.Time) string {
-	version := fmt.Sprintf("%d.%d", release.GetVersion().Major(), release.GetVersion().Minor())
-
-	return fmt.Sprintf("New module %s release %s is available. Release will be applied at: %s",
-		release.Spec.ModuleName, version, releaseApplyTime.Format(time.RFC850))
+func (webhookDataSource) Fill(output *updater.WebhookData, release *v1alpha1.ModuleRelease, applyTime time.Time) {
```

Comment:
I think here, as well as in the previous comment, it is worth providing nil checks for output and release parameters.

Just like describing the method on the pointer, since newWebhookDataSource() returns it.

Reply:
Fixed